api/discovery/aggregated_v2.json

```diff
@@ -1312,35 +1312,6 @@
             }
           ],
           "version": "v1beta1"
-        },
```

Comment:
these changes are unrelated and should be dropped ... make sure you have current tags fetched from upstream and regenerate API changes ... all of these changes should drop out